frontend/src/components/VSourcesTable.vue

```diff
@@ -111,6 +111,11 @@ export default defineComponent({
     function compareProviders(prov1: MediaProvider, prov2: MediaProvider) {
       let field1 = prov1[sorting.field]
       let field2 = prov2[sorting.field]
+      if (sorting.field === 'display_name' && typeof field1 === 'string' && typeof field2 === 'string') {
+        field1 = field1.toLowerCase()
+        field2 = field2.toLowerCase()
+      }
```

Comment:
I would prefer to have the same code as below: cast the value to a string instead of checking if the type is `string`:
```ts
field1 = (field1 as string).toLowerCase()
field2 = (field2 as string).toLowerCase()
```

Reply:
Alternatively, to avoid the case:

```suggestion
      if (sorting.field === 'display_name' && typeof field1 === 'string' && typeof field2 === 'string') {
        field1 = prov1[sorting.field].toLowerCase()
        field2 = prov2[sorting.field].toLowerCase()
      }
```